definitions/variable/technology/electricity-grid.yaml

```diff
@@ -28,13 +28,27 @@
 - Network|Electricity|Active Power Flow:
     description: Active power flow through a line
     unit: MW
+- Network|Electricity|Active Power Flow Generation Contribution:
+    description: The active power flow that a generator is causing on a line
+    unit: MW
+- Network|Electricity|Active Power Flow Demand Contribution:
```

Comment:
is this the share of the flow in interconnector region1>region2 which is caused by power demand in region3?